en/micro-integrator/docs/references/connectors/amazonsqs-connector/amazonsqs-inbound-endpoint-example.md

```diff
@@ -0,0 +1,106 @@
+           <parameter name="attributeNames">attributeName1,contentType</parameter>
+           <parameter name="contentType">application/json</parameter>
+           <parameter name="class">org.wso2.carbon.inbound.amazonsqs.AmazonSQSPollingConsumer</parameter>
+           <parameter name="inbound.behavior">polling</parameter>
+       </parameters>
+   </inboundEndpoint>   
+   ```   
+   **Sequence to process the message**
+   
+   In this example for simplicity we will just log the message, but in a real world use case, this can be any type of message mediation.
+ 
+   ```xml
+   <?xml version="1.0" encoding="UTF-8"?>
+   <sequence name="request" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+       <log level="full"/>
+   </sequence>
+   ```
+> **Note**: To configure the `secretKey` and `accessKey` parameter value, please use the [Setting up the Amazon Lambda Environment](../amazonlambda-connector/setting-up-amazonlambda.md) documentation.
+> - **secretKey** : The secret key used to sign requests.
+> - **accessKey** : The access key that corresponds to the secret key that you used to sign the request.
+> - **destination** : URL of the Amazon SQS Queue from which you want to consume messages.
+   
+## Exporting Integration Logic as a CApp
+
+**CApp (Carbon Application)** is the deployable artefact on the Enterprise Integrator runtime. Let us see how we can export integration logic we developed into a CApp. To export the `Solution Project` as a CApp, a `Composite Application Project` needs to be created. Usually, when a solution project is created, this project is automatically created by Integration Studio. If not, you can specifically create it by navigating to  **File** -> **New** -> **Other** -> **WSO2** -> **Distribution** -> **Composite Application Project**. 
+
+1. Right click on Composite Application Project and click on **Export Composite Application Project**.</br> 
+  <img src="../../../../assets/img/connectors/capp-project1.png" title="Export as a Carbon Application" width="300" alt="Export as a Carbon Application" />
+
+2. Select an **Export Destination** where you want to save the .car file. 
+
+3. In the next **Create a deployable CAR file** screen, select inbound endpoint and sequence artifacts and click **Finish**. The CApp will get created at the specified location provided in the previous step. 
+
+## Deployment
+
+1. Navigate to the [connector store](https://store.wso2.com/store/assets/esbconnector/list) and search for `AmazonSQS Connector`. Click on `AmazonSQS Inbound Endpoint` and download the .jar file by clicking on `Download Inbound Endpoint`. 
+   > **Note**: Copy this .jar file into **<PRODUCT-HOME>/dropins** folder.
+  
+2. Copy the exported carbon application to the **<PRODUCT-HOME>/repository/deployment/server/carbonapps** folder. 
+
+3. [Start the WSO2 EI server](https://ei.docs.wso2.com/en/latest/micro-integrator/overview/quick-start-guide/#start-the-micro-integrator). 
```

Comment:
What abt EI 6.6.0? @samgnaniah how we have put this in other inbound EP docs?

Reply:
We have used the latest docs and a relative link.

Reply:
Updated.